ui/src/main/webapp/src/app/project/migration-project.service.ts

```diff
@@ -132,6 +139,20 @@ export class MigrationProjectService extends AbstractService {
             .catch(this.handleError);
     }
 
+    @Cached('project', {minutes: 1})
+    getAllIncludingProvisional(): Observable<Array<MigrationProject & HasAppCount>> {
+        return this._http.get(Constants.REST_BASE + this.GET_MIGRATION_PROJECTS_URL + "?includeProvisional=true")
+            .map(res => <MigrationProjectAndCount[]> res.json())
+            .map(entries => entries.map(entry => (entry.migrationProject["applicationCount"] = entry.applicationCount, entry.migrationProject)))
+            .catch(this.handleError);
+    }
+
+    deleteProvisionalProjects(): Observable<void> {
+        return this._http.delete(Constants.REST_BASE + this.DELETE_PROVISIONAL_PROJECTS_URL)
```

Comment:
As explained on the meeting, it's the best way to implement it.
If you don't believe, you can take over the PR.